packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -1,22 +1,17 @@
-import { ChangeEvent, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import Blockies from "react-blockies";
 import { useEnsAddress, useEnsAvatar, useEnsName } from "wagmi";
 import { isAddress } from "ethers/lib/utils";
-
-type TAddressInputProps = {
-  onChange?: (arg: string) => void;
-  placeholder?: string;
-  name?: string;
-  value?: string;
-};
+import { InputBase } from "./InputBase";
+import { CommonInputProps } from "./utils";
 
 // ToDo:  move this function to an utility file
 const isENS = (address = "") => address.endsWith(".eth") || address.endsWith(".xyz");
 
 /**
  * Address input with ENS name resolution
  */
-const AddressInput = ({ value, name, placeholder, onChange }: TAddressInputProps) => {
+export const AddressInput = ({ value, name, placeholder, onChange }: CommonInputProps) => {
```

Comment:
I don't remember when but as I remember, @carletex mentioned that he prefers `export default` for components since we can use lazy loading in future

Reply:
Yeah there's an issue open about it, I still need to check out the details and update my PR accordingly :+1:

Reply:
I found the discussion (#118). The issue is with barrel files.
I removed all the barrel files and updated imports. I think we should keep named exports, as it will be (slightly) easier to add barrel files for a potential future scaffold-eth package if ever the nextjs issue is resolved.